pkgs/applications/misc/dupeguru/default.nix

```
@@ -0,0 +1,68 @@
+{stdenv, python3Packages, gettext, qt5, fetchFromGitHub}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "dupeguru";
+  version = "4.0.4";
+
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "arsenetar";
+    repo = "dupeguru";
+    rev = "${version}";
+    sha256 = "0ma4f1c6vmpz8gi4sdy43x1ik7wh42wayvk1iq520d3i714kfcpy";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    gettext
+    python3Packages.pyqt5
+    qt5.wrapQtAppsHook
+  ];
+
+  pythonPath = with python3Packages; [
+    pyqt5
+    send2trash
+    sphinx
+    polib
+    hsaudiotag3k
+  ];
+
+  buildFlags = [
+    "i18n"
+    "modules"
+    "qt/dg_rc.py"
+    "pyc"
+  ];
+
+  installFlags = [
+    "PREFIX=${placeholder ''out''}"
+    # As dupguru makefile is a little "broken" it installs python virtualenv on make install.
+    "--assume-old=all" 
+    "--assume-old=pyc"
+  ];
+
+  doCheck = false;
```

We could probably do

```
  checkInputs = with python3Packages; [
    pytest
    pytest-monkeyplus
  ];

  pytestFlagsArray = [ "core" "hscommon" ];
```

but that would require packaging `pytest-monkeyplus`.

If you do not feel like packaging it, please add a comment explaining that we are missing some test dependencies.

---

I think that it will be a nice task for another pr. I don't feel myself ready to do it right now.

---

Could you add it as a comment then?

```suggestion
  # TODO: package pytest-monkeyplus for running tests
  # https://github.com/NixOS/nixpkgs/pull/75054/files#r357690123
  doCheck = false;
```

---

@jtojnar fixed.